zarr-in-practice.ipynb

```diff
@@ -0,0 +1,3003 @@
+{
```

Comment:
<p>Do we want to structure this notebook as "high level for users" proceeding into "lower level for people with interest"? I.e. the specific contents of <code>.zarray</code> might not be that useful for most high level users. Maybe it would be more useful here to show the metadata from a Zarr dataset python object? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>